tools/send-to-caas/bulk-publish-to-caas.beta.js

```diff
@@ -127,7 +127,7 @@ const showSuccessTable = (successArr) => {
     tableBody.innerHTML += `<tr>
       <td>${++index}</td>
       <td class="ok">OK</td>
-      <td><a href="${pageUrl}" title="View page">${pageUrl}</a></td>
+      <td><a href="${pageUrl}" target="_blank" title="View page">${pageUrl}</a></td>
```

Comment:
This is a good usability improvement for the UI. Could add `rel=noopener noreferrer` so the new tab can't access `window.opener`